ts/components/IdpsGrid.tsx

```diff
@@ -80,10 +79,7 @@ const keyExtractor = (idp: LocalIdpsFallback): string => idp.id;
 // Image cache forced refresh for Android by appending
 // the `ts` query parameter as DDMMYYYY to simulate a 24h TTL.
 const androidIdpLogoForcedRefreshed = () => {
-  const timestampValue = localeDateFormat(
-    new Date(),
-    I18n.t("global.dateFormats.shortFormat").replace(/\//g, "")
-  );
+  const timestampValue = toAndroidCacheTimestamp();
   return Platform.OS === "android" ? `?ts=${timestampValue}` : "";
 };
```

Comment:
What do you think to refactor this as below:
```typescript
const androidIdpLogoForcedRefreshed = () => Platform.OS === "android" ? `?ts=${toAndroidCacheTimestamp()}` : "";
```

Reply:
Short and sweet, I like it. Thank you.